ci-operator/step-registry/ipi/gcp/pre/monitoringpvc/ipi-gcp-pre-monitoringpvc-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: ipi-gcp-pre-monitoringpvc
+  steps:
+  - chain: ipi-conf-gcp
+  - ref: ipi-install-monitoringpvc
+  - chain: ipi-install
```

Comment:
I'm not a big believer in the pre/post model, when these are so easy to inline in the workloads.  Or push this down?  Which consumers of ipi-install would not want this?

Reply:
This alters default openshift install - and some platforms may not be able to provide a PV

Reply:
So in that case, I'd push this up to the workflow without a pre chain.  But I don't have a problem with the pre chain if other folks want to maintain it.